### PR TITLE
refactor(all): fix typo

### PR DESCRIPTION
**Pull request by sashamelentyev**

### CODE REVIEW

File: schema/field/type.go [hunk 1]
```diff
@@ -73,7 +73,7 @@ func (t Type) ConstName() string {
 	switch {
 	case !t.Valid():
 		return typeNames[TypeInvalid]
-	case int(t) < len(constNames) && constNames[t] != "":
+	case t < len(constNames) && constNames[t] != "":
```
Comment (a8m) [inline schema/field/type.go:76]: ?

**Review by a8m — APPROVED** [review]
Thanks, @sashamelentyev for the contribution. Please fix the small lint error and I'll merge this.

**Comment by sashamelentyev** [comment]
@a8m, done.

**Review by a8m — APPROVED** [review]
Thanks ❤️

**Pull request merged**